components/WorkoutForm.js

```diff
@@ -0,0 +1,268 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { exercises } from "@/lib/exercises";
+import findExerciseById from "@/utils/findExerciseById";
+
+// Styled Components
+const Header = styled.header`
+  text-align: center;
+  margin-bottom: 1rem;
+`;
+
+const WrapperFilter = styled.div`
+  width: 100%;
+  height: 100%;
+  position: fixed;
+  top: 0;
+  z-index: 1;
+  background: rgba(0, 0, 0, 0.6);
+  padding: 1rem 3rem;
+`;
+
+const WrapperForm = styled.div`
```

Comment:
You don't really have to change this but this should probably be called FormWrapper because it's a wrapper for a form. WrapperForm made me assume that this itself should be a form element instead of a div. 😅